evergreen/compile-centos7-evergreen.sh

```diff
@@ -0,0 +1,13 @@
+ #!/bin/bash
+set -euo pipefail
```

Comment:
why not also add option 'x' ("Print commands and their arguments as they are executed.")

Reply:
(I see the compile-centos7.sh does already use -x)

Reply:
I think it's a good idea to add `x`. Done.